applications/GeoMechanicsApplication/custom_utilities/interface_element_utilities.hpp

```diff
@@ -93,49 +93,6 @@ class InterfaceElementUtilities
         rNu(2,14) = Ncontainer(GPoint,4); rNu(2,17) = Ncontainer(GPoint,5); rNu(2,20) = Ncontainer(GPoint,6); rNu(2,23) =  Ncontainer(GPoint,7);
     }
 
-    //----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-    static inline void CalculateNuElementMatrix(BoundedMatrix<double,3,12>& rNut,
-                                                const Matrix& Ncontainer,
-                                                const unsigned int& GPoint)
-    {
-        //Quadrilateral_interface_2d_4
-        rNut(0,0) = -Ncontainer(GPoint,0); rNut(0,3) = -Ncontainer(GPoint,1);
```

Comment:
Just for my understanding, what is the reason for the minus sign compared to the 'normal element' version?

Reply:
I actually don't know. Perhaps @WPK4FEM has an idea ...?

Reply:
At first glance minus on one side of the interface, plus on the other. Interfaces are about differential movement of the 2 sides.